riverdriver/river_driver_interface.go

```diff
@@ -100,76 +100,75 @@ type Executor interface {
 
 	// ColumnExists checks whether a column for a particular table exists for
 	// the schema in the current search schema.
-	ColumnExists(ctx context.Context, tableName, columnName string) (bool, error)
+	ColumnExists(ctx context.Context, params *ColumnExistsParams) (bool, error)
```

Comment:
Since everything was getting a new `schema` argument, I ended up just changing everything into a parameter struct so (1) we don't have some awkward functions that need to take three strings like this one, and (2) just so it's all predictable. Everything just takes a param struct.

This ended up requiring a lot of refactoring so not sure it was worth it in the end, but oh well.